mirroroperator/registrymirror.py

```diff
@@ -450,7 +477,14 @@ def generate_stateful_set(self):
                                             "memory": "500Mi"}
                                 ),
                                 volume_mounts=volumes_to_mount,
-                            )
+                            ),
+                            client.V1Container(
```

Comment:
Shouldn't this be an initContainer rather than a sidecar? Does it always need to be running?

Reply:
no resources specified for it

Reply:
I have made it an init container and given it the same resources as the main container.